code/web/src/modules/user/ProfileForm.js

```diff
@@ -37,8 +36,9 @@ class ProfileForm extends Component {
   }
 
   componentDidMount() {
-    let shippingAddress = '';
-    let description = '';
+    let shippingAddress = ''
+    let description = ''
+    let image = '/images/stock/default_profile.jpg'
     let userDetails; 
 
     if (this.props.user.details.address) {
```

Comment:
Noticed that this if statement is for the address and the inside is related to the description. We should probably fix this one before merging! @kncaputo

Reply:
I was wrong! GH doesn't show all the lines! We are good to go on this one. I believe we should be able to merge now! @kncaputo